ariadne/subscriptions.py

```diff
@@ -47,3 +47,15 @@ def bind_subscribers_to_graphql_type(self, graphql_type):
                 )
 
             graphql_type.fields[field].subscribe = subscriber
+
+
+class Subscription:
+    def __init__(
+        self,
+        operation_name: str,
+        async_generator: AsyncGenerator,
+        context_value: ContextValue,
+    ):
+        self.operation_name = operation_name
+        self.async_generator = async_generator
+        self.context_value = context_value
```

Comment:
Missing blank line at end of file:

```suggestion
        self.context_value = context_value

```